.github/workflows/quali.yml

```diff
@@ -497,12 +497,11 @@ jobs:
                   - -x
                   - -c
                   - |
-                    apt-get update
-                    apt-get install -y dnsutils
                     HOSTS="sec-${{ steps.reservation.outputs.identifier }}.cicd.lab.wlan.tip.build gw-${{ steps.reservation.outputs.identifier }}.cicd.lab.wlan.tip.build fms-${{ steps.reservation.outputs.identifier }}.cicd.lab.wlan.tip.build"
                     for HOST in \$HOSTS; do
-                      until sleep 1; getent hosts \$HOST; do :; done
-                      getent hosts \$HOST >> /etc/hosts
+                      HOST_IP=""
+                      until [[ ! -z "$HOST_IP" ]]; do HOST_IP=$(getent hosts $HOST); done;
+                      echo $HOST_IP >> /etc/hosts
```

Comment:
Should probably contain `echo "$HOST_IP $HOST"`, not just IP

Reply:
Good catch :+1: